GitCommands/Git/GitModule.cs

```diff
@@ -2547,17 +2547,20 @@ public IReadOnlyList<GitStash> GetStashes(bool noLocks = false)
                 ? GitCommandCache
                 : null;
 
-            bool nonZeroGitExitCode = firstId == ObjectId.WorkTreeId && secondId is not null && !isTracked;
             ExecutionResult result = await _gitExecutable.ExecuteAsync(
                 args,
                 cache: cache,
                 outputEncoding: LosslessEncoding,
-                throwOnErrorExit: !nonZeroGitExitCode);
+                throwOnErrorExit: false);
+            if (!result.ExitedSuccessfully)
+            {
+                return (patch: null, errorMessage: $"{result.StandardError}{Environment.NewLine}Git command: {args} ({result.ExitCode}){Environment.NewLine}");
```

Comment:
❔
```suggestion
                return (patch: null, errorMessage: $"{result.StandardError}{Environment.NewLine}exit code {result.ExitCode} from git {args}{Environment.NewLine}");
```

Reply:
I rather see the command first, the error code is not often interesting

Reply:
I like to be able to copy the complete git command including "git".
The meaning of the trailing " (n)" would not be very clear and would break the git command line if not omitted.

Reply:
adjusted